.circleci/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: centos:7 # We support centos 6, but 7 already has a cpanm package
+    steps:
+      - checkout
+      - run: 'yum install -y perl-App-cpanminus dpkg-dev fakeroot lsb-release gettext gcc'
```

Comment:
Don't we need `epel-release` first for `dpkg-dev` to be available?

Reply:
You're right, of course.

Apparently, `yum install -y` returns a 0 exit code as long as it was able to find at least one of the requested packages 😞

Reply:
Looks like I need to add the flag `--setopt=skip_missing_names_on_install=False`

Reply:
Yeah. This behavior was changed in DNF to a more reasonable one, it's quite annoying in YUM.